.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"flutter_downloader","dependencies":[]},{"name":"package_info","dependencies":[]},{"name":"path_provider","dependencies":[]},{"name":"permission_handler","dependencies":[]},{"name":"shared_preferences","dependencies":[]},{"name":"url_launcher","dependencies":[]},{"name":"webview_flutter","dependencies":[]}]}
```

Comment:
Please update `.gitignore` and remove this file from version control. It explicitly states that it should be excluded.